server/controllers/likeDislike.js

```diff
@@ -0,0 +1,227 @@
+import db from '../models';
```

Comment:
import { Article, LikesDislikes } from '../models';

Reply:
Thanks. I will do the needful